admin_guide/cluster_capacity.adoc

```diff
@@ -143,6 +143,12 @@ $ oc adm policy add-cluster-role-to-user cluster-capacity-role \
     system:serviceaccount:default:cluster-capacity-sa
 ----
 
+[NOTE]
+====
+The `default` specification in the above command refers to the `default`
```

Comment:
s/above/previous

> ! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”